examples/use_intelex.py

```diff
@@ -0,0 +1,222 @@
+"""
+Creating models that are accelerated by Intel(R) Extension for scikit-learn
+--------------------------------------------------------------------------------
+
+This guide demonstrates how Intel(R) Extension for scikit-learn (sklearnex) can
+be used to speed up inference of scikit-learn models.
```

Comment:
Maybe if there is a link to a document showing what models are supported, it would be nice to link it here.